src/objectExplorer/objectExplorerService.ts

```diff
@@ -575,6 +582,14 @@ export class ObjectExplorerService {
         return;
     }
 
+    private getParentNode(node: TreeNodeType): TreeNodeInfo {
```

Comment:
This doesn't need to be a member of the service, just have it be a module-level function outside the class.